lib/get-last-release.js

```diff
@@ -0,0 +1,40 @@
+const {promisify} = require('util');
+const {resolve} = require('url');
+const npmConf = require('npm-conf');
+const RegClient = require('npm-registry-client');
+const getClientConfig = require('./get-client-config');
+const getRegistry = require('./get-registry');
+
+module.exports = async ({publishConfig, name}, logger) => {
+  const config = npmConf();
+  const tag = (publishConfig || {}).tag || config.get('tag');
+  const {NPM_TOKEN, NPM_USERNAME, NPM_PASSWORD, NPM_EMAIL} = process.env;
+  const client = new RegClient(getClientConfig(config));
+  const registry = await getRegistry(publishConfig, name);
+
+  try {
+    const data = await promisify(client.get.bind(client))(resolve(registry, name.replace('/', '%2F')), {
+      auth: NPM_TOKEN ? {token: NPM_TOKEN} : {username: NPM_USERNAME, password: NPM_PASSWORD, email: NPM_EMAIL},
+    });
+    if (data && !data['dist-tags']) {
```

Comment:
why do you check for `data` her? Should this maybe be `!data || !data['dist-tags']`?

Reply:
This is something that come from the old plugin. It's to handle a weird case that happen on npm when the call return `data` but no `dist-tags`.
Normally when a package doesn't exist it returns a 404.
If I remember well this situation happens when you publish a version with a scope and you unpublish it, the registry return `{}` instead of a 404.